packages/playground-examples/copy/ko/TypeScript/Primitives/Literals.ts

```diff
@@ -0,0 +1,65 @@
+// TypeScript의 소스 코드에는 리터럴을 위한
+// 재미있고 특별한 기능들이 있습니다.
+
+// 부분적으로는, 타입 넓히기와 좁히기 ( example:type-widening-narrowing )가
+// 중점적으로 다뤄지고 있으므로
+// 그 부분을 먼저 살펴보도록 하곘습니다.
+
+// 리터럴은 collection 타입의 더욱 구체적인 하위 타입입니다.
```

Comment:
```suggestion
// 리터럴은 집합적인 타입의 더 구체적인 하위 타입입니다.
```
collection은 원문에 없는 내용 같아요.
집합적인도 어색한데 더 좋은 표현이 있을까요? :)

Reply:
보통도 컬렉션이라고 그냥 써서 한글로 번역한 말이 생각나지 않네요 🤔